### PR TITLE
upgrade prettier

### PR DESCRIPTION
**Pull request by wolverineks**
upgrade prettier
remove pretty lint

### CODE REVIEW

File: flow-typed/npm/@react-navigation/bottom-tabs_v5.x.x.js
Comment (v-almonacid) [inline flow-typed/npm/@react-navigation/bottom-tabs_v5.x.x.js]: Wouldn't it be better to ignore the `flow-type/` folder? (as well as any other third-party code?)

File: ios/emurgo/Images.xcassets/AppIcon-nightly.appiconset/Contents.json
Comment (v-almonacid) [inline ios/emurgo/Images.xcassets/AppIcon-nightly.appiconset/Contents.json]: I'd would remove everything under `ios/` too beacuse it may conflict with Xcode's internal linting, creating unnecessary code diffs in the future.

File: src/components/Common/LanguagePicker.js
Comment (v-almonacid) [inline src/components/Common/LanguagePicker.js:243]: I suspect this comment can be removed now, since prettier was added in eslint rules, it shouldn't conflict. There are a bunch of other cases

File: src/i18n/locales/en-US.json
Comment (v-almonacid) [inline src/i18n/locales/en-US.json]: Please remove also `src/i18n/locales/`, except maybe `/src/i18n/locales/**/en-US.json` and `/src/i18n/locales/**/en-US.md` which are the only ones that are edited manually. The rest is generated automatically and shouldn't be processed by prettier

**Review by v-almonacid — CHANGES_REQUESTED** [review]

File: src/i18n/locales/en-US.json
Comment (wolverineks) [reply src/i18n/locales/en-US.json]: not sure how to do this

File: flow-typed/npm/@react-navigation/bottom-tabs_v5.x.x.js
Comment (wolverineks) [reply flow-typed/npm/@react-navigation/bottom-tabs_v5.x.x.js]: done

File: ios/emurgo/Images.xcassets/AppIcon-nightly.appiconset/Contents.json
Comment (wolverineks) [reply ios/emurgo/Images.xcassets/AppIcon-nightly.appiconset/Contents.json]: done

File: src/components/Common/LanguagePicker.js
Comment (wolverineks) [reply src/components/Common/LanguagePicker.js:243]: done

File: src/i18n/locales/en-US.json
Comment (v-almonacid) [reply src/i18n/locales/en-US.json]: then let's just ignore the full `src/i18n/` folder, and `translations/` too 🙏

Comment (wolverineks) [reply src/i18n/locales/en-US.json]: done

**Comment by zuzunker** [comment]
@wolverineks, let's resolve conflicts here plz, and then we will merge right away 🙏 Sorry the PR got out of sync that much

**Comment by wolverineks** [comment]
removes dependency on prettylint
requires adding ci step `yarn prettier --list-different ./src/**/*.js`

**Comment by zuzunker** [comment]
@wolverineks , resolve conflicts please, after the hocs PR got merged

**Pull request closed without merging**
